source/definitions.rst

```diff
@@ -70,11 +68,8 @@ Definitions
 
        Designed to be used with NTS.
 
-   Maximo
-      STA’s support system for maintenance
-
    NTS
-      National Traffic management system at the :term:`STA`.
+      National Traffic management system in Sweden.
```

Comment:
I think it is better to write the full name "Swedish Tranport Administration", since several other municipalities in Sweden uses RSMP but not NTS.